core/packages/test/package.json

```diff
@@ -16,7 +16,7 @@
     "@polkadot/types": "9.6.2",
     "@polkadot/util": "10.1.11",
     "@polkadot/util-crypto": "10.1.11",
-    "@chainsafe/lodestar": "1.2.1",
+    "@chainsafe/lodestar": "^1.2.2",
```

Comment:
@yrong Does this bump fix the issues where the testnet was not starting up? @claravanstaden would it be better to pin lodestar to a specific version? I feel like pinning to a version will make troubleshooting environment start up issues easier.

Reply:
@alistair-singh @yrong yeah good idea, let's pin to `v1.2.2` since a lot can change within a version.

Reply:
@alistair-singh 
TBO not sure it is the root cause, but in one of my test env the beacon relayer return 404 error when [query finalized update](https://github.com/Snowfork/snowbridge/blob/c16aa0d63f7450440e1529d08761acd5a65ee9dc/relayer/relays/beacon/header/syncer/api.go#L505) with v1.2.1 while upgrading to v1.2.2 resolved, so I decide try to upgrade to v1.2.2.  Have you met this issue before and would appreciated if you guys can help to verify 1.2.2 is fine, then I am ok pin to this version.

Btw, the correct response should be something like this:

```
 curl http://localhost:9596/eth/v1/beacon/light_client/finality_update
{"data":{"attested_header":{"slot":"120","proposer_index":"0","parent_root":"0x66663ce62fb88b3e87b9ad85f4b57f5c0c1dc2c17f4eee26c7cf03003d0a1929","state_root":"0xde5434d3834b130d75cf77ea36792c4cdf544b9210606993db31f9bf6329b2a9","body_root":"0x96dd609f95b8c2950478fd4d4f5a8f10ce951797f8b32fabb39e83406dd6d2c5"},"finalized_header":{"slot":"104","proposer_index":"3","parent_root":"0x8457d1b603ba7aa9f9a330445b264521f4f26a88b05610ca17b9962271d54fd4","state_root":"0x73af2312436de3c3c749a861806c8401c30acc18bb5d35458c03cdced933b4ea","body_root":"0x70f2511e36524201177eddabb5a084d53e12671812899fe8fcd9d0d49f80e95e"},"finality_branch":["0x0d00000000000000000000000000000000000000000000000000000000000000","0x10c726fac935bf9657cc7476d3cfa7bedec5983dcfb59e8a7df6d0a619e108d7","0x2d032c327de032b41addfcce3addbee0e52706df98a8c880493b8e22de19b3c1","0xcc3a4a103f88b4631f9a1819340078938e503982bb6b98d9487963b5a1d2ce20","0x67fb2b9520445ff5f9fc9d84e94796b892144365cf693e3c7eb74ef969f1f60e","0x7d80e0deb52eaa4f5f409c377cdd3a29d0f50e081edc01ad89c4926c0d72aa97"],"sync_aggregate":{"sync_committee_bits":"0xffffffff","sync_committee_signature":"0xaa55cb9b92e23d995fb41e33c637b1ff45aa35802f2853253a8c6a8b3944798b93ae92d68257c03d99dfae400960893f05ba2207d37e6ba4240ebfb327927d267238ddfb9f81004ca886d03b062e379aaf2205852510f9c6349c61f9daf06c79"},"signature_slot":"121"}}
```

Reply:
Interesting @yrong! It seems like the same issue. I'm happy to pin on 1.2.2 and let's keep an eye on that issue if we see it again.